examples/error-cause/error_cause.md

```diff
@@ -0,0 +1,9 @@
+# Error cause
+
+Ballerina facilitates in also having a cause in an error value. 
```

Comment:
Let's not break the sentences unless we need to start a separate description. Let's fix all the descriptions.

Reply:
+1